my-app/src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import Board from './Board.js';
+import Square from './Square.js';
+
+
+
+
+
```

Comment:
Please, remove extra empty lines.
Imports need to be put in order. First we import React and all packages, then our internal components and only at the end the file with the style.